### DISCUSSION TITLE
Cannot connect to Github repository from Eclipse

### DISCUSSION THREAD

**Discussion by montarou** [community/community#38553] [Other Feature Feedback, Questions, & Ideas]
I try to push from Eclipse Git Repositories. When Push confirmation opens and ask me identification with user and Password. It failed because can't connect to any URL:https://github.com/xxx
I use the username and password that is used to connect to gitbach
Did someone have such problem

**Comment by david-engelmann** [comment]
Are you trying to connect from ssh. [If so your connection is an https remote. That's why the prompt has https:// in it. You'll need to change it to a remote that looks like git@github.com:user/repo.git instead. git remote set-url can do this, or you can edit .git/config directly if you are careful. -tpope](https://github.com/tpope/vim-fugitive/issues/2074).

If not you'll probably need to set up a Personal Access Token

**Comment by montarou** [comment]
I connect through http:// but finally i progress by setting Personal 
Access Token when modifying Develloper Settings


Le 08/11/2022 à 15:29, David Engelmann a écrit :
>
> Are you trying to connect from ssh. If so your connection is an https 
> remote. That's why the prompt has https:// in it. You'll need to 
> change it to a remote that looks like ***@***.***:user/repo.git 
> instead. git remote set-url can do this, or you can edit .git/config 
> directly if you are careful. -tpope 
> <https://github.com/tpope/vim-fugitive/issues/2074>.
>
> If not you'll probably need to set up a Personal Access Token
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/orgs/community/discussions/38553#discussioncomment-4087326>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AHA6SFUCU4LWODKIXD73IJLWHJPV3ANCNFSM6AAAAAAR2GYKEM>.
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>
>
--

**Reply by david-engelmann** [reply]
Yeah PAT works, SSH is best IMO
